src/transaction/boot_sr.c

```diff
@@ -2565,6 +2567,15 @@ boot_restart_server (THREAD_ENTRY * thread_p, bool print_restart, const char *db
   if (boot_Lob_path[0] != '\0')
     {
       error_code = es_init (boot_Lob_path);
+
+      if (error_code != NO_ERROR)
+	{
+	  goto error;
+	}
+
+      /* remove lob ces temp dir */
+      error_code = boot_lob_remove_temp_dir ();
```

Comment:
NIT:

boot_remove_all_temp_volumes() 함수와 네이밍 일관성을 가져가면 좋을 것 같습니다.

```suggestion
      error_code = boot_lob_remove_all_temp_dirs ();
```